### DISCUSSION TITLE
Elastic search Missing after update

### DISCUSSION THREAD

**Discussion by wolfmatt21** [Security-Onion-Solutions/securityonion#5476] [Unsupported Versions]
HI,

Some of my heavy nodes are missing elasticsearch and wondered if there was a fix. I have attached a screenshot below.

Im running 2.3.61.

![image](https://user-images.githubusercontent.com/82594848/132948321-2cb6223c-a34a-4d4c-927a-5b0c9562219f.png)

**Comment by dougburks** [comment]
Please check the Elasticsearch logs for additional clues:
https://docs.securityonion.net/en/2.3/elasticsearch.html#diagnostic-logging

**Reply by ghost** [reply]
+1 same situation here. After last update to SO 2.3.70
On distributed Setting, on Search Node.
Graphana Status Search Node, Process Status Fault
so-status -> so-elasticsearch -> Missing
After doing a salt .\ state.highstate
so-status -> so-elasticsearch -> Pending

From Master: 
sudo salt .\ state.highstate completes with 7 task failures on Search Node with several errors:

wazuhdir -> false -> FileExistsError: [Errno 17] File exists: '/nsm'
registertheagent -> false -> ERROR: Wazuh manager did not become ready after 30 attempts; unable to proceed with registration)
nsmdir -> false ->  [Errno 17] File exists: '/nsm'
so-elasticsearch -> docker_container_running -> False -> 'Error 500: error while creating mount source path '/nsm/elasticsearch': mkdir /nsm: file exists'.
importdir -> false -> [Errno 17] File exists: '/nsm'
nsmlsdir -> false -> [Errno 17] File exists: '/nsm'
run_module_setup -> false -> One or more requisite failed: elasticsearch.so-elasticsearch

After  state-highstate completes, so-status  -> so-elasticsearch  stattus goes back to Missing

No recent information on /opt/so/log/elasticsearch/<Search Node Hostname>.log (-7 days, log ends on Sept 14)

Thanks,

Edgar

**Reply by dougburks** [reply]
@edgarmtoro From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and simply provide a hyperlink to the related discussion.
